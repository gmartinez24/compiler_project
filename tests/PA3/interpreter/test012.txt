// Alex Garza - Assortment of boolean expression in combination with eachother 

main

bool a, b, e;
int c, d;

{
    a = false;
    b = call readBool();

    c = 4;
    d = call readInt();
    
    call printBool(c == d);

    call printBool((c != d) and b);

    call printBool((c < d) or (a or b));

    e = not((d > c) and (c > d));

    call printBool(e);
    
}.
