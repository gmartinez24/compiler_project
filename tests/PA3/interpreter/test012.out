bool? int? false true true true 